Extensions/blacklist.js

```diff
@@ -1174,6 +1178,287 @@ XKit.extensions.blacklist = new Object({
 		$("#xkit-extensions-panel-right").nanoScroller();
 		$("#xkit-extensions-panel-right").nanoScroller({ scroll: 'top' });
 
+		XKit.extensions.blacklist.nativeExportCpanel(m_div);
+	},
+
+	nativeExportCpanel: function(m_div) {
+
+		/**
+		 * Create elements with simple syntax
+		 *
+		 * @param {string} tagName - Type of element to create
+		 * @param {object} [attributes] - Property-value pairs to set as HTML/XML attributes (e.g. { href: '/' })
+		 * @param {object} [events] - Property-value pairs to set as event listeners (e.g. { click: () => {} })
+		 * @param {(Node|string)[]} [children] - Zero or more valid children
+		 * @returns {Element} Element created to specification
+		 */
+		const dom = (tagName, attributes = {}, events = {}, children = []) => {
+			const element = attributes && attributes.xmlns
+				? document.createElementNS(attributes.xmlns, tagName)
+				: document.createElement(tagName);
+
+			attributes && Object.entries(attributes).forEach(([name, value]) => element.setAttribute(name, value));
+			events && Object.entries(events).forEach(([type, listener]) => element.addEventListener(type, listener));
+			children && element.replaceChildren(...children);
+
+			element.normalize();
+			return element;
+		};
+
+		const apiFetch = async (resource, init) => {
+			return XKit.tools.async_add_function(
+				async ({ resource, init = {} }) => { // eslint-disable-line no-shadow
+					// add XKit header to all API requests
+					if (!init.headers) init.headers = {};
+					init.headers['X-XKit'] = '1';
```

Comment:
Depends if we want this fetch to show up in statistics along with XKit Rewritten's: https://github.com/AprilSylph/XKit-Rewritten/pull/529

Reply:
I think it makes more sense to continue using the same headers we use inside of New XKit instead of adopting the `X-XKit: 1` header from XRW (which gives us no way to distinguish between the two versions), but ultimately we should ask @cyle since he's the one consuming this information.

Reply:
yeah, please keep the two headers distinct. 

in fact, i didn't realize they were different in terms of header name, i thought the header _values_ were different. my bad for not catching that sooner. separate thing to look into later maybe. doesn't really matter too much as long as they're different somehow.

Reply:
Sure thing!

Of course, currently we don't set a header any other time that this version of XKit calls `window.tumblr.apiFetch` (it's only set on the legacy XHR helper), but I'll set this one to be consistent with what we have there.